src/lib/types/text.ts

```diff
@@ -14,12 +14,13 @@ export interface ITextProps extends IGeneralProps, TextProps {
   textAlign?: TextAlignType
   font?: string
   spacing?: number
-  family?: string
   o?: number
   /**
    * Utlity props
    */
   uppercase?: boolean
   capitalize?: boolean
   bold?: boolean
+  thin?: boolean
+  deco?: 'underline' | 'none' | 'through' | 'underline-through'
```

Comment:
These are invalid values for textDecorationLine

available values are: textDecorationLine: enum('none', 'underline', 'line-through', 'underline line-through')

https://facebook.github.io/react-native/docs/text#style

Reply:
This is now fixed!